cpp/ps0002/cpp_ps0002_solution.cpp

```diff
@@ -1,17 +1,18 @@
-#include<iostream>
+#include <iostream>
 using namespace std;
 int main()
 {
-	int population = 55;
-	short alternative = 3;
-	int members = 8;
-	float price = 5.5;
-	char letter = 'h';
-	double birth = 700000;
-	float div = 2.33;
-	float expenses = 1230.75;
-	double age = 10521000;
+	unsigned short population = 55; // 0.5/0.75
+	unsigned short alternative = 3; // 0.75/0.75
+	unsigned short members = 8; // 0.5/0.75
+	float price = 5.5; // 0.75/0.75
+	char letter = 'h'; // 0.75/0.75
+	unsigned int birth = 700000; // 0.5/0.75
+	long double div = 2.33; // 0.5/0.75
+	double expenses = 1230.75; // 0.5/0.75
+	unsigned int age = 10521000; // 0.5/0.75
+	// total 5.25/6.75

```

Comment:
- [x] all variables required in the document are declared
- [ ] variables are declared with the correct data type
- [x] using correct literals and notations for assigning initial values to the variables